proto/blockchain.proto

```diff
@@ -34,13 +34,16 @@ message NativeCallItem {
     uint32 id = 1;
     string method = 3;
     bytes payload = 4;
+    string nonce = 5;
 }
 
 message NativeCallReplyItem {
     uint32 id = 1;
     bool succeed = 2;
     bytes payload = 3;
     string errorMessage = 4;
+    string nonce = 5;
+    string signature = 6;
```

Comment:
Better to move it to a separate message that would have those two fields. Because I expect that this part is will be expanded later, and it will be easier to do in with a separate message.

Reply:
And I suggest to put the `signature` as raw bytes. Because otherwise  it's unclear how it's encoded to string, i.e. is it in hex or dec

Reply:
Ok, no problem